backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/impl/HUPickingSlotBL.java

```diff
@@ -655,4 +666,15 @@ public ImmutableList<HuId> retrieveAvailableHUIdsToPickForShipmentSchedule(@NonN
 		return retrieveAvailableHUIdsToPick(query);
 	}
 
+	public boolean clearPickingSlotQueue(@NonNull final PickingSlotId pickingSlotId, final boolean removeQueuedHUsFromSlot)
```

Comment:
consider making it private/package

Reply:
I can't as I'm implementing an interface